package.json

```diff
@@ -13,8 +13,10 @@
     "lint": "eslint '{src,test}/**/*.ts' --fix",
     "format": "prettier --write '{src,test}/**/*.ts'",
     "scrape": "tsx prisma/scraper.ts",
-    "scrape:scheduled": "SCHEDULED_MODE=true node dist/prisma/scraper.js",
-    "studio": "prisma studio"
+    "scrape:scheduled": "SCHEDULED_MODE=true tsx prisma/scraper.ts",
+    "studio": "prisma studio",
+    "send-notifications": "tsx prisma/send-notifications.ts",
+    "test-noti-setup": "tsx src/utils/test-notification-setup.ts"
```

Comment:
```test-notification-setup.ts``` would be a file in our tests (directory) if we had one. Since we don't, I think its safe to remove it (and this command) now and add it back if needed when we implement a test suite